mantle/cmd/ore/gcloud/upload.go

```diff
@@ -127,60 +128,55 @@ func runUpload(cmd *cobra.Command, args []string) {
 	}
 
 	imageStorageURL := fmt.Sprintf("https://storage.googleapis.com/%v/%v", uploadBucket, imageNameGS)
+	if uploadWriteUrl != "" {
+		err = ioutil.WriteFile(uploadWriteUrl, []byte(imageStorageURL), 0644)
+		if err != nil {
+			fmt.Fprintf(os.Stderr, "Writing file (%v) failed: %v\n", uploadWriteUrl, err)
+			os.Exit(1)
+		}
+	}
 
 	if uploadCreateImage {
 		fmt.Printf("Creating image in GCE: %v...\n", imageNameGCE)
 
-		// create image on gce
-		_, pending, err := api.CreateImage(&gcloud.ImageSpec{
-			Name:        imageNameGCE,
-			Family:      uploadImageFamily,
-			SourceImage: imageStorageURL,
-			Description: uploadImageDescription,
-		}, uploadForce)
-		if err == nil {
-			err = pending.Wait()
+		images, err := api.ListImages(context.Background(), imageNameGCE)
```

Comment:
This assumes that `ListImages` is strongly consistent.  On AWS, where it isn't, this change would absolutely introduce a race we'd occasionally lose.  I can't find any docs on the GCE consistency model, but would rather not introduce the potential race here.

Reply:
so I'm reading this as we should "change it back"?

Reply:
Yup.